solidity/random-beacon/contracts/RandomBeacon.sol

```diff
@@ -770,6 +770,20 @@ contract RandomBeacon is Ownable {
         }
     }
 
+    // TODO: Documentation.
+    function notifyFailedHeartbeat() external {
+        uint32[] memory ineligibleOperators;
+
+        // TODO: Determine ineligible operators.
```

Comment:
It should be passed in the param. It's up to the group to decide. These could be operators who failed an actual heartbeat  - were inactive during heartbeat signing - or operators who failed to submit result in their round.